meetings/2024-01-17.md

```diff
@@ -0,0 +1,161 @@
+* eP: We should add it as soon as possible in whatever form we reach rough consensus. Still at least list the possible issue. We can't expect everyone to be aware of the issues. It could be an inline issue to make people aware possible exists. And perhaps people give valuable feedback.
+* RG: Do we have like kanban board
+* SC: We have a project board: https://github.com/orgs/solid/projects/15/views/3
+* MdJ: We don't have a team of people working on it. 
+* MdJ: Please comment on the issue.
+
+
+### Access to Chairs on the Repo
+* eP: Do all chairs have access to solid/specification?
+* VB: I can take care of it.
+
+ACTION: VB to give access to co-chairs to solid/specification
+
+### Project board
+* RG: Should the project board be linked from ???
+* SC: Stuff that has been mostly active until last release been updating in some ways more towards milestoning for next release. Doesn't have categorization of more recent issues. That can be improved. Process those last issues into this board. They're not there. 
+* MdJ: If Sarven is not regularly grooming, then it's not good to link. Unless RG you want to pick it up yourself.
+* SC: Virginia will take care access to chairs and can say more on project board e.g. STM. 
+* VB: How are they related? Right for STM, not for issues. If that can be useful we can add more recent issues or talk about in these meetings.
+* MdJ: We shouldn't link to it until someone works on it.
+* HZ: I can help if we agree there is value.
+* VB: I can look into using the board as mentioned.
+* HZ: If we agree it has value I can groom it.
+* MdJ: I don't see particular value in it. There's more value if people want to do more work on the spec to do more PR reviewing, higher priority.
+* HZ: Agree but quite a few open issues.
+* VB: Agree with MDJ. Focus on PR review, and if time/resources are available.
+* MdJ: Can use these meetings as "our board".
+* eP: I'd support it. I can join HZ on this for next week discussion.
+* HZ: Sounds good.
+* RG: This is so that anybody who comes to this can have a centralised view which might make it easier.
+* MdJ: I'd be careful to not pick up work that we might not need. I'd say we try to work on the PRs and feel that not enough clarity and other issues to work, we can dig them up.
+
+
+#### Clarify requests with N3 document in server-representation-turtle-jsonld
+URL: https://github.com/solid/specification/pull/608
+
+* MdJ: This clarifies that when there is an N3 document, it should the same as RDF document. I suggested for the purposes of N3 patch
+* eP: I want to clarify the intent of this. In LDP we talk about (Non)RDF-Sources. Here we try to avoid that distinction but here we do. With N3 it happens to use an N3 document. The way to formulate may be to consider whether SPARQL Update may be used in the future. Take some time and see if we can formulate it in a way that's special for N3 but can also work with SPARQL. Also consideration on what's mature that can go to spec.
```

Comment:
```suggestion
* MdJ: This clarifies that when there is an N3 document, it should be the same as RDF document. I suggested for the purposes of N3 patch
* eP: I want to clarify the intent of this. In LDP, we talk about (Non)RDF-Sources. Here we try to avoid that distinction, but here we do ??? . With N3, it happens to use an N3 document. The way to formulate may be to consider whether SPARQL Update may be used in the future. Take some time and see if we can formulate it in a way that's special for N3 but can also work with SPARQL. Also consideration on what's mature that can go to spec.
```